plugins/psp-util.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  creationTimestamp: null
+  name: psp-util
+spec:
+  description: |
+    A Kubectl plugin to manage Pod Security Policy(PSP) and the related RBAC Resources.
+    Attach/Detach PSP to/from RBACs(Group, User) or ServiceAccounts and
+    view the relations which PSP is effected to the Subjects in cluster.
+  homepage: https://github.com/jlandowner/psp-util
+  platforms:
```

Comment:
I'm assuming you can probably also provide a windows build since it's written in go. No action needed for this PR.

Reply:
Thanks. I will provide a windows build in next version.